scripts/generate-go.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+set -e
+
+PROTOC=./bin/protoc/bin/protoc
+GO_OUTPUT=./go
+
+if ! [ -f "${PROTOC}" ]; then
+    echo "${PROTOC} not found! Please run scripts/install-protoc.sh first."
+    exit 1
+fi
+
+# TODO(zyy17): Can we make the following commands as one command?
+${PROTOC} --go_out=${GO_OUTPUT} --go_opt=paths=source_relative \
+          --go-grpc_out=${GO_OUTPUT}  --go-grpc_opt=paths=source_relative \
+          -Iproto proto/greptime/v1/meta/*.proto
+
+${PROTOC} --go_out=${GO_OUTPUT}  --go_opt=paths=source_relative \
+          --go-grpc_out=${GO_OUTPUT}  --go-grpc_opt=paths=source_relative \
+          -Iproto proto/greptime/v1/*.proto
+
+${PROTOC} --go_out=${GO_OUTPUT}  --go_opt=paths=source_relative \
+          --go-grpc_out=${GO_OUTPUT}  --go-grpc_opt=paths=source_relative \
+          -Iproto proto/prometheus/remote/*.proto
```

Comment:
Is it helpful to generate the code via temporary docker container? like https://github.com/namely/docker-protoc

Reply:
It's excellent advice! I will change to your advice later.

Reply:
Or maybe we can make this progress more automatical? It helps if the updates of proto files is just followed by the updates of generated files in this repo. Github Actions may help.

Reply:
yeap, it's also very helpful, but I think we can make it in the next PR(GitHub Action is very very hard to debug...).

Reply:
@yuanbohan When I try `namely/docker-protoc`, I think it's more complicated than what I thought. To make everything more simple and controllable, I think the fixed-version binary will be better.

Reply:
@yuanbohan, my mistake, I figure out how to use `namely/docker-protoc`. I decide to use it.